routing.md

```diff
@@ -277,9 +277,35 @@ Why is such indirection useful?
 * It lets the mediator remain uncommitted to whether the next receiver is another mediator or not. This may provide flexibility in some routing scenarios.
 * It lets the mediator change the size of the message by adding or subtracting noise from the content. 
 * It allows for dynamic routing late in the delivery chain.
- 
+
 These last two characteristics are the foundation of mix networking feature for DIDComm. That feature is the subject of a different RFC; here we only note the existence of the optional feature. 
 
 
-### DID Docs and Routing
+### DID Document Service Endpoint
+DIDComm DID Document endpoints have the following format:
+
+```json
+{
+    "id": "did:example:123456789abcdefghi#didcomm-1",
+    "type": "DIDComm",
+    "publicKeys": ["did:example:123456789abcdefghi#key-1"],
+    "serviceEndpoint": "http://example.com/path",
+    "routingKeys": ["did:example:somemediator#somekey"]
+}
+```
+
+**id**: must be unique, as required in [DID Core](https://www.w3.org/TR/did-core/#service-endpoints). No special meaning should be inferred from the `id` chosen.
+
+**type**: MUST be `DIDComm`. 
+
+**publicKey**s: MUST be an array of strings, to specify the keys to be used when encrypting a message to be sent to this endpoint.
+
+**serviceEndpoint**: MUST contain a URI for a transport specified in the [transports] section of this spec.
+
+**routingKeys**: An optional list of keys to be used when preparing the message for transmission as specified in the [Routing] section of this spec.
```

Comment:
Is this still an ordered list?

Reply:
It MUST be. Otherwise the correct order for wrapping cannot be determined. And this requires a custom JSON-LD context, since JSON-LD insists on viewing arrays as unordered by default (contrary to the JSON spec).

Reply:
(this can have considerations for JSON-LD DID documents.)